dataCompareR/tests/testthat/testCheckPrintObject.R

```diff
@@ -131,18 +131,18 @@ test_that("print rcomp object", {
   # For now we won't hard code each - instead, we will just check a few points...
   
   # We should look for the all match label
-  expect_that(any(textSame == "All compared variables match "),is_true())
-  expect_that(any(textSame == " Number of rows compared: 150 "),is_true())
-  expect_that(any(textSame == " Number of columns compared: 5"),is_true())
+  expect_true(any(textSame == "All compared variables match "),is_true())
```

Comment:
These aren't throwing warnings anymore, but I believe the `is_true()` and `is_false()` below should probably be removed as well. I'm happy to make that change myself after merging though!